data/data-pipeline/data_pipeline/etl/score/etl_score.py

```diff
@@ -509,6 +517,10 @@ def _prepare_initial_df(self) -> pd.DataFrame:
         #     For *Expected Agricultural Loss*, we only want to include in the percentile tracts
         #     in which there is some agricultural value. This helps us adjust the data such that we have
         #     the ability to discern which tracts truly are at the 90th percentile, since many tracts have 0 value.
+        #
```

Comment:
Right now, I want to make sure we have a quick and easy way to update and change this. Because we only get "share of land" from TPL, after we ground truth, I want to leave the door open to "left clip" this data.